docs/plugins/geant4.md

```diff
@@ -32,3 +34,50 @@ Possible implementations of this type of conversions can be seen in the correspo
   BOOST_CHECK(surface->type() == Acts::Surface::SurfaceType::Cylinder);
 ```
 
+#### Inspecting surface conversion within python
+
+The `ActsExampels` python bindings allow to conveniently test the conversion of `Geant4` volumes into sensitive and passive surfaces, assume you have a GDML file called `detector.gdml` where `Geant4PhysVolume` objects can be identified by a certain string, e.g. names containing the flag `Sensitive`, or `Passive`. Also multiple match strings are allowed. The converted surfaces can then be displayed with `.obj` (part of the Core functionality) or as `.svg` files (if `ACTS_BUILD_PLUGIN_ACTSVG` is switched on)
+
+```python
+  # import the necessary modules
+  import acts, acts.examples
+  from acts.examples import geant4 as acts_g4
+  
+  # The match criteria
+  sensitive_matches = [ 'Sensitive' ]
+  passive_matches = [ 'Passive' ]
+  [ elements, ssurfaces, psurfaces ] = acts_g4.convertSurfaces('detector.gdml', sensitive_matches, passive_matches)
+
+  # Some screen output
+  print('* Conversion yielded', len(ssurfaces))
+
+  # Write them to an obj file 
+  drawContext = acts.GeometryContext()
+  sensitiveRgb = [ 0, 150, 150 ]
+  passiveRgb = [ 150, 150, 0]
+  # Draw the sensitive surfaces
+  acts.examples.writeSurfacesObj(ssurfaces, drawContext, sensitiveRgb, 'detector-sensitives.obj')
+  # Draw the passive surfaces
+  acts.examples.writeSurfacesObj(psurfaces, drawContext, passiveRgb, 'detector-passives.obj')
+```
+
+## Building a Detector from Geant4 input
+
+In order to build a `Acts::Detector` object from `Geant4` input, the following steps needs to be done
+ * a conversion of `Geant4PhysVolume` objects into `Acts::Surface`  and `Acts::DetectorVolume` objects (see before)
+ * a build sequence needs to be defined and the converted objects identified
+
+ There are several helper methods and tools that can be used, many of them accessible through python bindings, one core component is the selection and assignment of surfaces to dedicated volume. This can be done using e.g. a KDT structure, this can be tested with:
```

Comment:
```suggestion
 There are several helper methods and tools that can be used, many of them accessible through python bindings. One core component is the selection and assignment of surfaces to dedicated volume. This can be done using e.g. a KDT structure, this can be tested with:
```

Reply:
Updated.